### PR TITLE
Add some new spaceport messages.

### PR DESCRIPTION
**Pull request by petervdmeer**
**Content (New human news messages)**

## Summary
Added some new human spaceport news messages.

## Save File
N/A

## PR Checklist
N/A

### CODE REVIEW

File: data/human/news.txt [hunk 1]
```diff
@@ -365,6 +373,7 @@ news "groundskeeper"
 			"They told me that they wanted the landscaping to be classic and entertaining, so I constructed a giant hedge maze."
 			"Despite all our best terraforming efforts, it can still be hard to get native Earth plants to grow on an alien world."
 			"My employer once asked me to add a garden in their Star Queen. The pirates were sure confused when they boarded us."
+			"My employer owns almost 2 complete planets, but today I'm watering the plans in the small appartment she owns on this planet."
```
Comment (Zitchas) [inline data/human/news.txt:376]: Did you mean to say she is watering the plans here, or should that actually be "plants"? Also mis-spelling of apartment.

```suggestion
			"My employer owns almost 2 complete planets, but today I'm watering the plants in the small apartment she owns on this planet."
```

File: data/human/news.txt [hunk 2]
```diff
@@ -241,6 +242,7 @@ news "company psychologist"
 			"I've gotten incredibly good at doing this thing with my face that makes it look like I care."
 			"A trouble shared is a trouble halved, so most resource issues can be brought within acceptable limits by just pretending to listen."
 			"A man is judged by his deeds, not by his words, except in my profession; the more I talk, the more people value me."
+			"People are sometimes affraid that we psyhologists can read their minds, but we are actually more scared of learning our own motivations during our studies."
```
Comment (Corraban2) [inline data/human/news.txt:245]:
```suggestion
			"People are sometimes afraid that we psychologists can read their minds, but we are actually more scared of learning our own motivations during our studies."
```

File: data/human/news.txt [hunk 3]
```diff
@@ -303,6 +308,8 @@ news "union organizer"
 			"There's more than enough wealth to go around. All that we're asking for is our fair share."
 			"Syndicate managers get paid three times as much as their employees, just for sitting in an office all day and watching us sweat."
 			"My job is to encourage workers to place a higher value on their own lives despite being surrounded by a culture that tells them they are worthless."
+			"Management warning workers not to join the union is one of our best sources of visiblity among the workers."
+			"Young people usually don't join an union, so we are mostly working for the benefit of higher aged workers."
```
Comment (Corraban2) [inline data/human/news.txt:312]:
```suggestion
			"Young people usually don't join a union, so we are mostly working for the benefit of higher aged workers."
```

File: data/human/news.txt [hunk 4]
```diff
@@ -344,6 +351,7 @@ news "housekeeper"
 			"How is it possible that a man who manages a billion-credit corporation can still be so incompetent or so careless that he routinely misses the toilet bowl when he takes a piss?"
 			"The family I work for offered to let me live in a room in the attic for free, but I'm not sure I want to have that little separation between work and my personal life."
 			"If rich kids are so pampered that they've never so much as had to wash a dish in their entire lives, it makes me wonder how they hope to be independent adults one day."
+			"The kids here are not spoiled at all; the kids in the family I work for typically don't get a new computergame until two days after it gets on the market."
```
Comment (Corraban2) [inline data/human/news.txt:354]:
```suggestion
			"The kids here are not spoiled at all; the kids in the family I work for typically don't get a new computer game until two days after it gets on the market."
```

File: data/human/news.txt [hunk 5]
```diff
@@ -284,6 +287,8 @@ news "unemployed youth"
 			"Actually, I'm working on a novel. Well, right now I'm just doing world-building and research, so I haven't written much of the actual story yet."
 			"Do you know how lucky you are, that you can visit a different world every week if you want to? I've been stuck on this one filthy little planet my whole life."
 			"If I had enough money, I'd go up to a captain in the spaceport and ask for transport to anywhere else. Nowhere's as bad as this place."
+			"Are you hiring new crewmembers? And are you by chance planning to fly to the paradise worlds?"
```
Comment (Terin) [inline data/human/news.txt:290]:
```suggestion
			"Are you hiring new crewmembers? And are you by chance planning to fly to the Paradise Worlds?"
```

File: data/human/news.txt [hunk 6]
```diff
@@ -284,6 +287,8 @@ news "unemployed youth"
...
+			"We had to stay at home during a pandemic years ago. It was horrible, the walls of my appartment made me crazy."
```
Comment (Amazinite) [inline data/human/news.txt:291]: Eh. Not feeling putting this one in.

File: data/human/news.txt [hunk 7]
```diff
@@ -365,6 +373,7 @@ news "groundskeeper"
 			"They told me that they wanted the landscaping to be classic and entertaining, so I constructed a giant hedge maze."
 			"Despite all our best terraforming efforts, it can still be hard to get native Earth plants to grow on an alien world."
 			"My employer once asked me to add a garden in their Star Queen. The pirates were sure confused when they boarded us."
+			"My employer owns almost 2 complete planets, but today I'm watering the plants in the small apartment she owns on this planet."
```
Comment (Amazinite) [inline data/human/news.txt:376]:
```suggestion
			"My employer owns almost two complete planets, but today I'm watering the plants in the small apartment she owns on this planet."
```

File: data/human/news.txt [hunk 8]
```diff
@@ -423,6 +432,7 @@ news "human resources worker"
 			"We always put our employees first. But sometimes they do things that are bad for their careers. Like, accusing a high-ranking manager of harassment."
 			"I once had to find a delicate way to explain to a worker that he smelled bad. Like, so bad that no one could bear to be in a small meeting room with him."
 			"For most jobs, it's essential to know how to get along with others and be a team player. But I bet a freelance pilot like you never needs to worry about that."
+			"Most people overestimate or underestimate their own capabilities. The people who overestimate their capabilities usually don't last long in their jobs."
```
Comment (Amazinite) [inline data/human/news.txt:435]: What about people who underestimate their own capabilities?

**Review by Amazinite — CHANGES_REQUESTED** [review]

File: data/human/news.txt (hunk 6 above)
Comment (petervdmeer) [reply data/human/news.txt:291]: Good point. Might be too close to home when the whole word is in a pandemic.

File: data/human/news.txt (hunk 8 above)
Comment (petervdmeer) [reply data/human/news.txt:435]: I'll split this one into two versions.

**Review by Amazinite — APPROVED** [review]

**Pull request merged**